src/core/public/chrome/ui/header/collapsible_nav_group_enabled.tsx

```diff
@@ -0,0 +1,347 @@
+import { createEuiListItem } from './nav_link';
+import type { Logos } from '../../../../common/types';
+import {
+  ChromeNavGroupServiceStartContract,
+  ChromeRegistrationNavLink,
+  NavGroupItemInMap,
+} from '../../nav_group';
+import {
+  fulfillRegistrationLinksToChromeNavLinks,
+  getOrderedLinksOrCategories,
+  LinkItem,
+  LinkItemType,
+} from '../../utils';
+import { ALL_USE_CASE_ID } from '../../../../../core/utils';
+import { CollapsibleNavTop } from './collapsible_nav_group_enabled_top';
+import { HeaderNavControls } from './header_nav_controls';
+
+export interface CollapsibleNavGroupEnabledProps {
+  appId$: InternalApplicationStart['currentAppId$'];
+  basePath: HttpStart['basePath'];
+  id: string;
+  isLocked: boolean;
+  isNavOpen: boolean;
+  navLinks$: Rx.Observable<ChromeNavLink[]>;
+  storage?: Storage;
+  onIsLockedUpdate: OnIsLockedUpdate;
+  closeNav: () => void;
+  navigateToApp: InternalApplicationStart['navigateToApp'];
+  navigateToUrl: InternalApplicationStart['navigateToUrl'];
+  customNavLink$: Rx.Observable<ChromeNavLink | undefined>;
+  logos: Logos;
+  navGroupsMap$: Rx.Observable<Record<string, NavGroupItemInMap>>;
+  navControlsLeftBottom$: Rx.Observable<readonly ChromeNavControl[]>;
+  currentNavGroup$: Rx.Observable<NavGroupItemInMap | undefined>;
+  setCurrentNavGroup: ChromeNavGroupServiceStartContract['setCurrentNavGroup'];
+}
+
+interface NavGroupsProps {
+  navLinks: ChromeNavLink[];
+  suffix?: React.ReactElement;
+  style?: React.CSSProperties;
+  appId?: string;
+  navigateToApp: InternalApplicationStart['navigateToApp'];
+  onNavItemClick: (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    navItem: ChromeNavLink
+  ) => void;
+}
+
+const titleForSeeAll = i18n.translate('core.ui.primaryNav.seeAllLabel', {
+  defaultMessage: 'See all...',
+});
+
+export function NavGroups({
+  navLinks,
+  suffix,
+  style,
+  appId,
+  navigateToApp,
+  onNavItemClick,
+}: NavGroupsProps) {
+  const createNavItem = ({
+    link,
+    className,
+  }: {
+    link: ChromeNavLink;
+    className?: string;
+  }): EuiSideNavItemType<{}> => {
+    const euiListItem = createEuiListItem({
+      link,
+      appId,
+      dataTestSubj: `collapsibleNavAppLink-${link.id}`,
+      navigateToApp,
+      onClick: (event) => {
+        onNavItemClick(event, link);
+      },
+    });
+
+    return {
+      id: `${link.id}-${link.title}`,
+      name: <EuiText size="s">{link.title}</EuiText>,
+      onClick: euiListItem.onClick,
+      href: euiListItem.href,
+      emphasize: euiListItem.isActive,
+      className: `nav-link-item ${className || ''}`,
+      buttonClassName: 'nav-link-item-btn',
+      'data-test-subj': euiListItem['data-test-subj'],
+      'aria-label': link.title,
+    };
+  };
+  const createSideNavItem = (navLink: LinkItem, className?: string): EuiSideNavItemType<{}> => {
+    if (navLink.itemType === LinkItemType.LINK) {
+      if (navLink.link.title === titleForSeeAll) {
```

Comment:
what does see all mean? Is it a toggle/switch between all vs collapsed?

Reply:
<img width="631" alt="image" src="https://github.com/user-attachments/assets/1efac059-e84e-4d19-aa49-7bfb669e1b95">
So `See all...` will go deep into the use case.